api/v1alpha1/dataset_types.go

```diff
@@ -353,7 +353,7 @@ func (dataset *Dataset) SetDataOperationInProgress(operationType string, name st
 		}
 		return
 	}
-	if dataset.Status.OperationRef[operationType] == "" {
+	if dataset.Status.OperationRef[operationType] == "" || strings.Contains(dataset.Status.OperationRef[operationType], name) {
```

Comment:
`strings.Contains` may not be accurate when there are multiple cron DataMigrate with similar name. I think we can split the string and check it.

Reply:
done